awx/main/models/credential/__init__.py

```diff
@@ -1092,6 +1096,62 @@ def create(self):
     },
 )
 
+ManagedCredentialType(
+    namespace='openshift_username_password',
+    kind='kubernetes',
+    name=ugettext_noop('OpenShift Username / Password'),
+    managed_by_tower=True,
```

Comment:
We shouldnt be passing this keyword argument as it [won't be used](https://github.com/ansible/awx/blob/devel/awx/main/models/credential/__init__.py#L581)

I _do_ see that some of the other MangedCredentialTypes are already doing this as well, but those can be resolved in a different pr / issue as it doesn't directly relate to this PR.